strategies/SimplePatienceStrategyV2.py

```diff
@@ -0,0 +1,32 @@
+from .IterativeStrategy import IterativeStrategy
+
+
+class SimplePatienceStrategyV2(IterativeStrategy):
+    """
+    Patience Equation: p(x) = p
+    """
+
+    _name = "simple_patience_v2"
+    _short_name = "SPv2"
+
+    def __init__(self, b: int = 0, **kwargs):
+        super().__init__(**kwargs)
+        self.b = b
+
+    @property
+    def patience(self):
```

Comment:
Previously, I was supporting the use of a parameter `patience` for simple patience, I might want to transition to simply using `b` though. Let me know if you think I should keep the original `parameter` argument though.

Reply:
`b` would be better I think.